### PR TITLE
feat: Add alert on project settings for users without permission

### PR DESCRIPTION
**Pull request by lynnagara**
If user does not have project:write access do not try to load view and display alert.

Fixes #JAVASCRIPT-2XZ and others.

### CODE REVIEW

**Comment by lynnagara** [comment]
Words by @MaxBittker :)

**Review by ckj — APPROVED** [review]
LGTM.

File: src/sentry/static/sentry/app/views/projectSettings/index.jsx [hunk 1]
```diff
@@ -170,11 +171,22 @@ const ProjectSettings = createReactClass({
           </ul>
         </div>
         <div className="col-md-10">
-          {React.cloneElement(this.props.children, {
-            setProjectNavSection: this.props.setProjectNavSection,
-            project,
-            organization: this.context.organization,
-          })}
+          {access.has('project:write') ? (
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/projectSettings/index.jsx]: are there any settings inside this page that they're allowed to see? the code looks good to me

**Review by MaxBittker — APPROVED** [review]

File: src/sentry/static/sentry/app/views/projectSettings/index.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/projectSettings/index.jsx]: No, pretty sure the whole section is only supposed to be visible for users with write access

**Pull request merged**